### PR TITLE
labours: update relative Swivel import

### LINKED ISSUE #9: Coupling calculations fails on Swivel import
Under the virtualenv

```
go get gopkg.in/src-d/hercules.v2/cmd/hercules
virtualenv -p python3 .venv
pip3 install -r requirements.txt
hercules -couples https://github.com/src-d/go-git | python3 labours.py -m couples -o gogit_couples.tsv
```

Fails with

```
Reading the input... done
Traceback (most recent call last):
  File "labours.py", line 679, in <module>
    sys.exit(main())
  File "labours.py", line 650, in main
    *train_embeddings(files_coocc, args.couples_tmp_dir))
  File "labours.py", line 421, in train_embeddings
    from . import swivel
ImportError: cannot import name 'swivel'
```

### PR DESCRIPTION
**Pull request by bzz**
This fixes #9 

\cc @vmarkovtsev for review

### CODE REVIEW

File: labours.py [hunk 1]
```diff
@@ -419,8 +419,9 @@ def train_embeddings(coocc_tree, tmpdir, shard_size=4096):
     from scipy.sparse import csr_matrix
     try:
         from . import swivel
-    except SystemError:
+    except ImportError:
```
Comment (vmarkovtsev) [inline labours.py]: Python2.7 user? Py3.x raises SystemError.
```
except (SystemError, ImportError)
```

Comment (bzz) [inline labours.py]:
```
python --version
Python 3.6.1
```

Comment (bzz) [inline labours.py]: From https://docs.python.org/3/reference/import.html
> Loaders must satisfy the following requirements:
> If the loader cannot execute the module, it should raise an ImportError

Comment (bzz) [inline labours.py]: Thanks for the tip, history re-written at 8a03b56

**Comment by bzz** [comment]
Thanks for the tip, review addressed and history re-written at 8a03b56

File: labours.py (hunk 1 above)
Comment (vmarkovtsev) [inline labours.py]:
```
In [4]: from . import test_artm_model
---------------------------------------------------------------------------
SystemError                               Traceback (most recent call last)
<ipython-input-4-e6adabd80afa> in <module>()
----> 1 from . import test_artm_model

SystemError: Parent module '' not loaded, cannot perform relative import
```

This was changed in 3.6: https://github.com/python/cpython/blob/8a9cd20edca7d01b68292036029ae3735ce65edd/Misc/NEWS.d/next/Core%20and%20Builtins/2017-07-11-06-31-32.bpo-30876.x35jZX.rst

**Comment by vmarkovtsev** [comment]
Thanks! Somehow my IDE showed the `tensorflow` import as unused and I trusted that bastard.

**Pull request merged**